### ISSUE TITLE
No data being sent

### ISSUE THREAD

**Issue by jamiekarvans** [Blank-c/Umbral-Stealer#9]
hello, no data is being sent to webhook,
have you changed any links maybe or injection links?

**Comment by Blank-c** [comment]
I haven't. Check if your webhook is okay, try to build it without vmprotect option, turn off antivirus too.

**Comment by jamiekarvans** [comment]
yes thanx, it was the vmprotect, for some reason it was causing the problem
